db/models/app.model.js

```diff
@@ -27,3 +27,16 @@ exports.fetchArticleById = (article_id) => {
       }
     });
 };
+
+exports.fetchComments = (article_id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE comments.article_id = $1 ORDER BY comments.created_at DESC;`, [
+      article_id,
+    ])
+    .then((results) => {
```

Comment:
This looks odd? A message is being thrown of "ID not" and is not used by the error handler. Your error handlers also look odd, next is called directly after a res.status().send() which wont have any effect as the response will already be sent. Your 500 error handler will never currently and your app will respond with a 404 as its default error.